src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -6669,7 +6669,7 @@ public void TrailingCommaInPropertyPattern_02()
                 N(SyntaxKind.IsKeyword);
                 N(SyntaxKind.RecursivePattern);
                 {
-                    N(SyntaxKind.PropertyPatternClause);
+                    N(SyntaxKind.ListPatternClause);
```

Comment:
Please add a corresponding test (missing patterns in list-pattern) for LangVer "preview"

Reply:
Sould that be reported during parsing?

Reply:
Sould that be reported during parsing?

Reply:
> Should that be reported during parsing?

That's not necessary. The missing name-colon in property sub-pattern is reported during binding for example. I think we can do the same for this scenario.

Reply:
We already observe langveraion behavior in `ListPattern_01` (no error).